images.CI/linux-and-win/lint_ubuntu.ps1

```diff
@@ -0,0 +1,34 @@
+param(
+    [Parameter(Mandatory)] [string] $path,
+    [Parameter(Mandatory)] [string] $pattern
+)
+
+$ErrorActionPreference = "Stop"
+
+function Validate-Scripts {
+    Param (
+        [Parameter(Mandatory=$true)]
+        [string[]]$Path,
+        [Parameter(Mandatory=$true)]
+        [string]$Pattern
+    )
+     $WrongScript = New-Object System.Collections.Generic.List[System.Object]
+     Get-ChildItem $path | ForEach-Object {
+       if (Select-String -Path $($_.FullName) -Pattern $Pattern -Quiet) {
```

Comment:
It is poor validation.
If script contains shebang but it is located at the last line of script, it won't help us.
I think we should do `Get-Content` and check the first line of script

Reply:
Thank you, I've changed.